modules/data.remote/R/LandTrendr.AGB.R

```diff
@@ -183,6 +264,8 @@ download.LandTrendr.AGB <- function(outdir, target_dataset = "biomass", product_
     results$formatname[i] <- out_formats[i]
   }
   
+  suppressWarnings(PEcAn.DB::db.close(con, showWarnings = FALSE)) # why isnt TRUE invisible? 
```

Comment:
Please remove this line. Functions should  not close db connections that they were passed as arguments (and thus that they didn't open themselves) because they don't know if that connection will be needed for a later function.